dvc/command/version.py

```diff
@@ -46,13 +45,11 @@ def run(self):
             # `dvc config cache.shared group`.
             if os.path.exists(repo.cache.local.cache_dir):
                 info.append(
-                    "Cache: {}".format(self.get_linktype_support_info(repo))
+                    "Cache types: " + self.get_linktype_support_info(repo)
                 )
                 if psutil:
                     fs_type = self.get_fs_type(repo.cache.local.cache_dir)
-                    info.append(
-                        f"Filesystem type (cache directory): {fs_type}"
-                    )
+                    info.append(f"Cache with {fs_type}")
```

Comment:
I agree with @pmrowla that it looks strange with mixed colon/no-colon output. Let's just use `Cache directory: ext4 on /dev/sda4` kind of output. Same for workspace too (Workspace directory?).